lib/facade_converter.ts

```diff
@@ -244,6 +244,16 @@ export class FacadeConverter extends base.TranspilerBase {
         this.emitCall('remove', c.arguments);
         this.emit('!= null || true ) )');
       },
+      'Map.forEach': (c: ts.CallExpression, context: ts.Expression) => {
+        const callback = <ts.FunctionExpression>c.arguments[0];
```

Comment:
What about the case of passing an already bound expression? `map.forEach(myFn)`? I think it's ok to given an error, but we should have a proper message.